yarn-project/circuits.js/src/tests/factories.ts

```diff
@@ -72,6 +73,8 @@ import { SignedTxRequest, TxRequest } from '../structs/tx_request.js';
 import { CommitmentMap, G1AffineElement, VerificationKey } from '../structs/verification_key.js';
 import { range } from '../utils/jsUtils.js';
 import { numToUInt32BE } from '../utils/serialize.js';
+import { computeCallStackItemHash } from '../abis/abis.js';
+import { WasmWrapper } from '@aztec/foundation/wasm';
```

Comment:
can this import be added to the other `@aztec/foundation` imports at the top of the file?

Reply:
shockingly no - it seems like it isn't exported into `@aztec/foundation`

Reply:
should we add it?

Reply:
Foundation exports some modules under a different folder, see https://github.com/AztecProtocol/aztec3-packages/blob/master/yarn-project/foundation/package.json#L28